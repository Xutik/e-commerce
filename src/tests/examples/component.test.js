const renderGreeting = require("../../assets/js/testExamples/component")

describe("testing component render", () => {
    test("match the snapshot", () => {
      const output = renderGreeting('FED26');
      expect(output).toMatchSnapshot();
    });
  });