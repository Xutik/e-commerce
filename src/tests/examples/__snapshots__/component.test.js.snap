// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing component render match the snapshot 1`] = `"<h1>Hello, FED26!</h1>"`;
